webpack.dev.ts

```diff
@@ -59,26 +61,41 @@ const config: webpack.Configuration = merge(common, {
     new ConfigurationPlugin({
       filename: 'index.ejs',
       outputFilename: 'index.html',
-      overrides: computeEndpoints(),
-    }),
-    new ProxyPlugin([
-      {
-        service: 'api',
-        localPort: 8000,
-      },
-      {
-        service: 'happaapi',
-        localPort: 8888,
-      },
-      {
-        service: 'passage',
-        localPort: 5001,
-      },
-      {
-        service: 'oidc_issuer',
-        localPort: 9999,
+      overrides: {
+        apiEndpoint: 'http://localhost:8000',
+        mapiEndpoint: 'http://localhost:8888',
+        mapiAudience: 'http://localhost:9999',
+        passageEndpoint: 'http://localhost:5001',
+        mapiAuthRedirectURL: 'http://localhost:7000',
+        environment: 'development',
+        happaVersion: 'development',
+        enableRealUserMonitoring: false,
       },
-    ]),
+      proxies: [
```

Comment:
Here we configure all the proxies necessary for bypassing CORS requirements on backend endpoints.